frontend/apps/racing-car-game/src/features/Game/component/Layout/Layout.tsx

```diff
@@ -56,14 +61,14 @@ function LayoutComponent() {
 
   const getDecodedPayload = (payload: Bytes) => {
     if (meta?.types.others.output) {
-      return meta.createType(meta?.types.others.output, payload).toHuman();
+      return meta.createType(meta?.types.others.output, [null, payload]).toHuman();
```

Comment:
```suggestion
      return meta.createType(meta?.types.others.output, [null, payload]).toHuman() as [null, DecodedReply];
```
I think better to make assertion locally in a function